assets/stylesheets/emergent-paywall.scss

```diff
@@ -0,0 +1,687 @@
+// The Emergent Paywall loads in an iframe, so we can't adjust its internal styles
+// with the primary Calypso stylesheet. This file is bundled as a separate CSS file to be
+// linked from inside the iframe.
+//
+// Emergent requires that all custom CSS files exist at a public web address
+// So the CSS files Directlyhard-set to:
+//
+// Sandbox:    https://wpcalypso.wordpress.com/calypso/emergent-paywall.css
+// Production: https://wordpress.com/calypso/emegent-paywall.css
+//
+// ## DEVELOPMENT ENVIRONMENT INSTRUCTIONS:
+//
+// To see local CSS changes you'll need to
+// make your browser rewrite those CSS URLs to the local version:
+//
+// https://.wordpress.com/calypso/emergent-paywall.css -> http://calypso.localhost:3000/calypso/emergent-paywall.css
+//
+// You can accomplish this using a browser extension such as Switcharoo Redirector
+// for Chrome: https://chrome.google.com/webstore/detail/switcheroo-redirector/cnmciclhnghalnpfhhleggldniplelbg
+//
+// An unfortunate side-effect is that the widget's iframe src is https, but this local
+// CSS file is http. So most browsers will block the CSS by default to prevent mixed
+// content. In order to see the custom CSS you will need to temporarily allow insecure
+// content on the page. In Chrome for Mac you can issue this Terminal command:
+//
+// open -a Google\ Chrome --args --allow-running-insecure-content
+//
+// QUIT AND RESTART YOUR BROWSER NORMALLY WHEN YOU'RE DONE MAKING LOCAL CSS CHANGES.
+
+@import 'shared/mixins/mixins';
+@import 'shared/colors';
+@import 'shared/forms';
+@import 'shared/typography';
+@import 'shared/functions';
+
+* {
+	font-family: $sans;
+}
+
+body {
+	color: $gray-dark;
+	-webkit-font-smoothing: subpixel-antialiased;
+}
+
+html,
+input,
+select,
+textarea {
+	font-family: $sans;
+}
+
+.body-hosted {
+	min-width: 562px; // we allow it to grow for embedded off site iframe
+	min-height: 100px;
+	border: none;
+	background-color: $white;
+
+	.header {
+		display: none;
+	}
+
+	// Seen in the 2nd step, displays the payment method type
+	.header2, .method-header {
+		margin: 0 0 20px 0;
+
+		strong, span {
+			font-size: 24px !important;
+			font-weight: 300;
+		}
+
+		img {
+			max-height: 20px;
+			margin-left: 10px;
+		}
+	}
+
+	.row {
+		height: 300px;
+
+		.btn-tile-cont {
+			float: left;
+			margin: 10px 0;
+			width: 50%;
+			height: 100px;
+
+			.btn-tile {
+				text-align:center;
+				cursor: pointer;
+				padding: 10px;
+
+				img {
+					max-width: 220px;
```

Comment:
At `220px`, the PayTM images overlap at lower viewport widths. Anything at or below `max-width: 180px;` seems to work.